src/components/PersonalData/PersonalData.tsx

```diff
@@ -1,30 +1,28 @@
 import React from 'react'
-import { GridContainer, Grid } from '@trussworks/react-uswds'
+import { Grid } from '@trussworks/react-uswds'
 import styles from './PersonalData.module.scss'
 
 const PersonalData = () => {
   return (
     <div className={styles.personalData}>
-      <GridContainer>
```

Comment:
@abbyoung just FYI, I removed `<GridContainer>` from this component because (misleadingly) it's not a required container for Grid components. It's used to apply margins/centering for the whole page content and isn't meant to be nested.